velox/exec/SetAccumulator.h

```diff
@@ -86,6 +102,37 @@ struct SetAccumulator {
     }
   }
 
+  void readNullIndex(common::InputByteStream& stream) {
```

Comment:
Let's make sure that only serialize/deserialize methods are public and the rest are private.

Reply:
These (read, serialize and isNullIndex) are not private since StringViewSetAccumulator/ComplexTypeSetAccumulator use those methods in their serialize/deserialize logic.

Reply:
Got it. Can these be protected?

Reply:
The current code is using containment and not inheritance so we cannot use protected either.